### DISCUSSION TITLE
Do not notify draft PR has merge conflicts

### DISCUSSION THREAD

**Discussion by alexanderwu-db** [community/community#129660] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

I have notifications turned on for PR status (the Slack integration: Integrations -> Scheduled reminders). It notifies me when my PR has merge conflicts. This feature isn't that useful in the first place as I usually already know, or will check it and find out eventually. But it even notifies me for draft PRs. If it's a draft, clearly it's not ready for merge anyway, so I should not care about merge conflicts IMO. I would highly prefer if we were not notified about merge conflicts on draft PRs.

**Comment by marc-hb** [comment]
This is likely a request for Slack more than for Github.

BTW I wish conflict notification worked "out of the box" (as in Gitlab?) and did not require Slack in the first place.
- https://github.com/isaacs/github/issues/224

Also interesting analysis:
- https://github.com/pullreminders/backlog/issues/42#issuecomment-436412823
